Olha_Lesiuk/HW_7/task_2.py

```diff
@@ -0,0 +1,6 @@
+def square(x):
+    return x**2, x*4
+
+
+a, b = square(int(input()))
+print(str(a) + "is area", str(b) + "is perimeter")
```

Comment:
read the task more carefully. It is not a solution

Reply:
resolved